service-tests/github.js

```diff
@@ -205,3 +205,87 @@ t.create('hit counter for nonexistent repo')
     name: Joi.equal('goto counter'),
     value: Joi.string().regex(/^repo not found$/),
   }));
+
+t.create('open issues')
+  .get('/issues/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]? open$/)
+  }));
+
+t.create('open issues (raw)')
+  .get('/issues-raw/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]?$/)
+  }));
+
+t.create('open pull requests')
+  .get('/issues-pr/cdnjs/cdnjs.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('pull requests'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]? open$/)
+  }));
+
+t.create('open pull requests (raw)')
+  .get('/issues-pr-raw/cdnjs/cdnjs.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('pull requests'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]?$/)
+  }));
+
+t.create('closed issues')
+  .get('/issues-closed/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('closed issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]? closed$/)
+  }));
```

Comment:
It seems to me that **closed** is not needed here, since the left side already says it's counting closed issues. Then we could omit `-closed-raw` as well.

Reply:
Rights, I think this was done because it's closer to what we do for open. Otherwise there's no difference anymore between `closed-raw` and `closed`.

Reply:
almost my entire interest here is in closed issues (though i do not understand or care about the `-raw`) distinction

Reply:
Haha. Ok, I'll do what you suggested.